src/VideoPlayer.xml

```diff
@@ -0,0 +1,103 @@
+    </icon>
+    <properties>
+        <property key="urlAttribute" type="attribute" required="false">
+            <caption>Video URL</caption>
+            <category>General</category>
+            <description>A full URL of the video; Youtube, Vimeo, Dailymotion or MP4.</description>
+            <attributeTypes>
+                <attributeType name="String"/>
+            </attributeTypes>
+        </property>
+        <property key="urlStatic" type="string" required="false">
+            <caption>Static URL</caption>
+            <category>General</category>
+            <description>A static url ensures your video will show when there is no value provided from an attribute.
+                When there is a URL provided by an attribute, that will override the static URL.
+            </description>
+        </property>
+        <property key="posterAttribute" type="attribute" required="false">
+            <caption>Poster URL</caption>
+            <category>General</category>
+            <description>The poster image is a custom preview image that will be shown in the player until the user
+                starts the video.
+            </description>
+            <attributeTypes>
+                <attributeType name="String"/>
+            </attributeTypes>
+        </property>
+        <property key="posterImageUrl" type="string" required="false">
+            <caption>Poster Static URL</caption>
+            <category>General</category>
+            <description/>
+        </property>
+        <property key="autoStart" type="boolean" defaultValue="false">
+            <caption>Auto start</caption>
+            <category>Behavior</category>
+            <description>Automatically start playing the video when the page loads</description>
+        </property>
+        <property key="showControls" type="boolean" defaultValue="true">
```

Comment:
Can we do this better way with the new group functions (we dont need to release it till WM is fixed)

Reply:
Done, please review again